purchase_open_qty/tests/test_purchase_open_qty.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import TransactionCase
+from openerp.fields import Datetime
+
+
+class TestPurchaseOpenQty(TransactionCase):
+    def setUp(self):
+        super(TestPurchaseOpenQty, self).setUp()
+        self.purchase_order_model = self.env['purchase.order']
+        purchase_order_line_model = self.env['purchase.order.line']
+        partner_model = self.env['res.partner']
+        prod_model = self.env['product.product']
+        analytic_account_model = self.env['account.analytic.account']
+        self.product_uom_model = self.env['product.uom']
+
+        pa_dict = {
+            'name': 'Partner 1',
+            'supplier': True,
+        }
+        self.partner = partner_model.sudo().create(pa_dict)
+        po_dict = {
+            'partner_id': self.partner.id,
+        }
+        # PURCHASE ORDER NUM 1
+        self.purchase_order_1 = self.purchase_order_model.create(po_dict)
+        uom_id = self.product_uom_model.search([
+            ('name', '=', 'Unit(s)')])[0].id
+        pr_dict = {
+            'name': 'Product Test',
+            'uom_id': uom_id,
+        }
+        self.product = prod_model.sudo().create(pr_dict)
+        ac_dict = {
+            'name': 'analytic account 1',
+        }
+        self.analytic_account_1 = \
+            analytic_account_model.sudo().create(ac_dict)
+        pl_dict1 = {
+            'date_planned': Datetime.now(),
+            'name': 'PO01',
+            'order_id': self.purchase_order_1.id,
+            'product_id': self.product.id,
+            'product_uom': uom_id,
+            'price_unit': 1,
```

Comment:
I will be converted to float but for more consistency use directly a float: `1.0`